### PR TITLE
embed links

### PR DESCRIPTION
**Pull request by Darqam**
add links to base doc page as well as command link. Note that setTitle was swapped out for setDescription to allow for markup linking.

### CODE REVIEW

File: src/commands/info/docs.js [hunk 1]
```diff
@@ -65,28 +65,26 @@ class DocsCommand extends Command {
         return embed;
     }
 
-    formatProp(item, mainItem) {
+    formatProp(item, mainItem, version) {
         const embed = this.client.util.embed();
 
-        const title = `${mainItem.name}${item.scope === 'static' ? '.' : '#'}${item.name}`;
-        embed.setTitle(title);
+        let description = `[${mainItem.name}${item.scope === 'static' ? '.' : '#'}${item.name}](https://discord.js.org/#/docs/main/${version}/class/${mainItem.name}${item.scope === 'static' ? '.' : '?scrollTo='}${item.name})`;
```
Comment (1Computer1) [inline src/commands/info/docs.js]: These links are incorrect for static members, they look like `scrollTo=s-PROPERTY`

**Review by 1Computer1 — CHANGES_REQUESTED** [review]

File: src/commands/info/docs.js [hunk 2]
```diff
@@ -42,14 +42,14 @@ class DocsCommand extends Command {
...
         const embed = this.client.util.embed();
 
-        let title = item.name;
-        if (item.extends) title += ` (extends ${item.extends[0]})`;
-        embed.setTitle(title);
+        let description = `[${item.name}](https://discord.js.org/#/docs/main/${version}/class/${item.name})`;
+        if (item.extends) description += ` (extends ${item.extends[0]})`;
 
-        if (item.description) embed.setDescription(this.clean(item.description));
+        if (item.description) description += `\r\n${this.clean(item.description)}`;
```
Comment (1Computer1) [inline src/commands/info/docs.js]: Why a carriage return here (and in other places)? That's not needed.

File: src/commands/info/docs.js [hunk 3]
```diff
@@ -98,9 +96,9 @@ class DocsCommand extends Command {
         }
 
         if (item.returns) {
-            const description = item.returns.description ? `${this.clean(item.returns.description)}\n` : '';
+            const descript = item.returns.description ? `${this.clean(item.returns.description)}\n` : '';
```
Comment (1Computer1) [inline src/commands/info/docs.js]: Change this back to `description` to be consistent with the rest of the methods.

**Review by 1Computer1 — CHANGES_REQUESTED** [review]

File: src/commands/info/docs.js (hunk 3 above)
Comment (Darqam) [reply src/commands/info/docs.js]: The issue with this one is `description` is used in the upper scope on line 84. That was why I changed it to a different word, but maybe I should just change the other descriptions to another term?

Comment (1Computer1) [reply src/commands/info/docs.js]: Oh that is true, `desc` sounds nicer, go with that :P

**Review by 1Computer1 — APPROVED** [review]

**Pull request merged**